cmd/oras/internal/option/remote.go

```diff
@@ -113,10 +116,16 @@ func (opts *Remote) Parse() error {
 	return opts.distributionSpec.Parse()
 }
 
-// readPassword tries to read password with optional cmd prompt.
+// readPassword tries to read password and identity-token with optional cmd prompt.
```

Comment:
NIT: make line length less than 80
```suggestion
// readPassword tries to read password and identity-token with optional cmd
// prompt.
```